static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js

```diff
@@ -0,0 +1,79 @@
+define([
+    'commercial/modules/commercial-features',
+    'common/modules/commercial/contributions-utilities',
+    'lib/ajax',
+    'lib/config',
+    'lib/cookies',
+    'lib/storage',
+    'lodash/utilities/template',
+    'raw-loader!common/views/contributions-epic-single-button.html'
+], function (
+    commercialFeatures,
+    contributionsUtilities,
+    ajax,
+    config,
+    cookies,
+    store,
+    template,
+    contributionsEpicSingleButton
+) {
+
+    function canBeDisplayed() {
```

Comment:
You can get rid of this as the only difference to the default checks in  `contributions-utilities` is the sensitive check, which you can control with `showForSensitive` in the constructor object.